react/getting_started_with_react/what_is_jsx/what_is_jsx.md

```diff
@@ -0,0 +1,234 @@
+### Introduction
+
+You're likely somewhat familiar with JSX from previous React lessons, in this lesson we're going to explain _what_ it is, _why_ we use it, and _how_ to use it in your React apps.
+
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- What JSX is.
+- Differences between JSX and HTML.
+
+### What is JSX?
+
+JSX is a syntax extension for Javascript that lets you write HTML-like markup inside a Javascript file. It's not required to use JSX when writing React components, but it does make writing them more simple and concise.
+
+Essentially, JSX is syntactic sugar for the React [createElement](https://reactjs.org/docs/react-api.html#createelement) function. This function creates a React element, which is a plain object, so JSX compiles down to plain JavaScript objects.
+
+Here you can see the logged value of a div JSX element with some text:
+
+![JSX element's logged value](./imgs/00.png)
```

Comment:
Just leaving this comment here so we don't forget.

After the lesson is merged, this image link needs to be replaced with a statically link. Our guide on [adding images in the curriculum](https://github.com/TheOdinProject/curriculum/wiki/Adding-Images-to-the-Curriculum) can be followed.